Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -193,13 +193,13 @@ theorem rnDeriv_restrict (μ ν : Measure α) [SigmaFinite μ] [SigmaFinite ν]
     {s : Set α} (hs : MeasurableSet s) :
     (μ.restrict s).rnDeriv ν =ᵐ[ν] s.indicator (μ.rnDeriv ν) := by
   rw [← withDensity_indicator_one hs]
-  refine (rnDeriv_withDensity_left ?_ ?_ ?_).trans (ae_of_all _ (fun x ↦ ?_))
+  refine (rnDeriv_withDensity_left ?_ ?_ ?_).trans ?_
```

Comment:
this change (+ its twin at line 202) seem dubious (but the one in between is good)